push_notifications/webpush.py

```diff
@@ -0,0 +1,41 @@
+from . import NotificationError
+from .conf import get_manager
+
+
+class WebPushError(NotificationError):
+	pass
+
+
+def get_subscription_info(application_id, uri, browser, auth, p256dh):
+	url = get_manager().get_browser_post_url(application_id, browser)
+	return {
+		"endpoint": "%s/%s" % (url, uri),
+		"keys": {
+			"auth": auth,
+			"p256dh": p256dh
+		}
+	}
+
+
+def webpush_send_message(uri, message, browser, auth, p256dh, application_id=None, **kwargs):
+	try:
+		from pywebpush import webpush
+		from pywebpush import WebPushException
+	except ImportError:
+		raise WebPushError("Please install pywebpush package: pip install pywebpush")
```

Comment:
This should not be a WebPushError. In fact, you can let the ImportError happen. If you'd like for this to be checked for, you can implement a django check that looks at your settings and checks the correct lib is present.

Reply:
Same behaviour then apns2 [e443eb2b8d44b0f7e8079cec53b50afb10d1e365]